### ISSUE TITLE
Weird error returned

### ISSUE THREAD

**Issue by mateusap1** [mateusap1/nano-pos#5]
```
(node:11065) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 message-from-worker listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit
```
Normally happens after changing the settings a lot times

**Comment by mateusap1** [comment]
Now it happened while I was changing "windows"

**Comment by mateusap1** [comment]
Apparently, it happens when I update the information for the second time

**Comment by mateusap1** [comment]
Only happens when I click on "Receive Payments"

**Comment by mateusap1** [comment]
Fixed it on f4dcdaa26ecc8f66913d76c025be4ea37dcd7216 by removing `ipcRenderer.on()` from receivePayments component
